dlc_developer_config.toml

```diff
@@ -74,7 +74,7 @@ ec2_tests_on_heavy_instances = false
 sagemaker_local_tests = false
 
 # run standard sagemaker remote tests from test/sagemaker_tests
-sagemaker_remote_tests = false
```

Comment:
I think you also need to add a new buildpec (refer to the old buildspec-neuronx.yaml in its change history) and make changes under [buildspec_override] to use that buildspec to have the pipeline pick up the correct transformers version.

Reply:
thanks JJ, I changed the buildspec file, however its still failing the checks

Reply:
It seems like the image is not pushed. Where is the pr hook trying to look up the pre-push image and when/where is it supposed to be built/pushed?